bin/build.sh

```diff
@@ -60,7 +66,10 @@ echo "Using CC=${CC} and CXX=${CXX} compilers for building."
 
 mkdir -p build 
 cd build && cmake \
+                -DCMAKE_C_COMPILER=${CC} \
```

Comment:
Passing these here directly so that there is no confusion when someone has both GCC and clang.